src/login.rs

```diff
@@ -0,0 +1,4 @@
+pub struct Login {
+    pub username: String,
+    pub password: String,
+}
```

Comment:
Do you think the same struct will be used in the HTTP API? If so, maybe we need a module & folder like `entities` for these kinds of shared structs. Otherwise, maybe put it under `service`.

Reply:
I'm not sure about the HTTP API yet. I'll put it to `services` and then we can later move it to a separate folder/module if necessary.